### PR TITLE
Add Czech translation

### PR DESCRIPTION
**Pull request by p-bo**

### CODE REVIEW

File: data/translations/kdiskmark_cs_CZ.ts [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="265"/>
+        <source>SEQ 2</source>
+        <translation>SEQ 2</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="326"/>
+        <source>RND 1</source>
+        <translation>NHD 1</translation>
```
Comment (JonMagon) [inline data/translations/kdiskmark_cs_CZ.ts:162]: That's not good idea to translate RND

File: data/translations/kdiskmark_cs_CZ.ts [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="213"/>
+        <source>Block Size</source>
+        <translation>Velikost bloku</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="229"/>
+        <source>Queues</source>
+        <translation>Front</translation>
```
Comment (JonMagon) [inline data/translations/kdiskmark_cs_CZ.ts:421]: Maybe it is better to use nominativ in plural, front**y**?

File: data/translations/kdiskmark_cs_CZ.ts [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="245"/>
+        <source>Threads</source>
+        <translation>Vláken</translation>
```
Comment (JonMagon) [inline data/translations/kdiskmark_cs_CZ.ts:426]: Vlákna?

Comment (p-bo) [reply data/translations/kdiskmark_cs_CZ.ts:426]: It is about inflexing - supposing there sentence "How many threads?" (in Czech "Kolik vláken?")

File: data/translations/kdiskmark_cs_CZ.ts (hunk 2 above)
Comment (p-bo) [reply data/translations/kdiskmark_cs_CZ.ts:421]: It is about inflexing - supposing there sentence "How many queues?" (in Czech "Kolik front?")

File: data/translations/kdiskmark_cs_CZ.ts (hunk 3 above)
Comment (JonMagon) [reply data/translations/kdiskmark_cs_CZ.ts:426]: Ok

File: data/translations/kdiskmark_cs_CZ.ts (hunk 1 above)
Comment (p-bo) [reply data/translations/kdiskmark_cs_CZ.ts:162]: Maybe you are right - tried to be comprehensible there, but must admin, that this abbreviation is not used somewhere else. Thus reverting to English one.

**Pull request merged**